project3/styles.css

```diff
@@ -0,0 +1,381 @@
+    top: 6px;
+    height: 24px;
+    width: 24px;
+    margin-right: 10px;
+}
+
+input[type="radio"] {
+    border-radius: 50%;
+}
+
+input[type="radio"]:focus,
+input[type="checkbox"]:focus,
+input[type="radio"]:hover,
+input[type="checkbox"]:hover {
+    border: 2px solid var(--color-green);
+}
+
+input[type="radio"]:checked,
+input[type="checkbox"]:checked {
+    border: 2px solid var(--color-offwhite);
+    background-color: var(--color-green);
+}
+
+
+input[type="radio"]:checked+label {
+    padding-bottom: 2px solid var(--color-green);
+}
+
+input[type="text"],
+input[type="email"],
+input[type="number"] {
+    margin-bottom: 25px;
+}
+
+
+input[type="text"]:focus,
+input[type="email"]:focus,
+input[type="number"]:focus {
+    border-bottom: 3px solid var(--color-green);
+}
+
+textarea:focus {
+    border: 3px solid var(--color-green);
+}
+
+
+span {
+    display: inline-block;
+    position: relative;
+    left: 30px;
+    top: -33px;
+    margin-left: 5px;
+    width: 95%;
+}
+
+input[type="text"]:not(:placeholder-shown):invalid,
+input[type="email"]:not(:placeholder-shown):invalid,
+input[type="number"]:not(:placeholder-shown):invalid {
+    border-bottom-color: var(--color-red);
+}
+
+
+/*  ================== GLOBAL SETTINGS ================== */
+
+.age {
+    display: block;
+}
+
+.button-reset {
+    background: var(--color-green);
+    border: 5px solid var(--color-offblack);
+    cursor: pointer;
+    display: block;
+    font-family: 'Teko', sans-serif;
+    font-size: 32px;
+    font-weight: 300;
+    height: 90px;
+    margin: auto;
+    width: 200px;
+    padding: 0;
+    position: relative;
+    text-align: center;
+    text-transform: uppercase;
+    top: 50px;
+}
+
+.button-reset:hover {
+    background: var(--color-green);
+    border: 5px solid var(--color-offblack);
+    cursor: pointer;
+    display: block;
+    font-family: 'Teko', sans-serif;
+    font-size: 32px;
+    font-weight: 300;
+    height: 90px;
+    margin: auto;
+    width: 200px;
+    outline: 3px solid var(--color-green);
+    padding: 0;
+    text-align: center;
+}
+
+.button-reset:active {
+    background: var(--color-offwhite);
+    border: 5px solid var(--color-green);
+    color: var(--color-offblack);
+    cursor: pointer;
+    display: block;
+    font-family: 'Teko', sans-serif;
+    font-size: 32px;
+    font-weight: 300;
+    height: 90px;
+    margin: auto;
+    width: 200px;
+    outline: 3px solid var(--color-green);
+    padding: 0;
+    text-align: center;
+}
+
+​ .button-reset[disabled] {
+    cursor: default;
+}
+
+.box {
+    border-bottom: 80px solid var(--color-offblack);
+    background-color: var(--color-offblack);
+    height: fit-content;
+    padding-left: 25px;
+    padding-right: 25px;
+    position: relative;
+    margin: auto;
+    min-width: 350px;
+    top: 80px;
+    width: 60%;
+}
+
+.boxes {
+    margin: auto;
+}
+
+.catagory {
+    margin-right: 30px;
+}
+
+.catagory,
+.gender {
+    display: inline-block;
```

Comment:
This is duplicating some of the CSS you already wrote for the `fieldset` element.